src/main/java/tdd/setup/Calculator.java

```diff
@@ -9,28 +10,27 @@
 
     private String latestOperation = "";
 
-    public String readScreen() {
+    public String readScreen()
+    {
```

Comment:
Das mit den geschweiften Klammern in einer neuen Zeile finde ich sehr gewöhnungsbedüftig und zählt nicht zu den Best Practices in Sachen Code Style